code/__DEFINES/bitflags.dm

```diff
@@ -0,0 +1,3 @@
+#define BITSHIFT_LEFT(X, N) (X << (N))
+#define BITSHIFT_RIGHT(X, N) (X >> (N))
+#define BITFLAG(X) BITSHIFT_LEFT(1, X)
```

Comment:
why do we need these defines?

bitshifts are a  pretty fundamental thing
i don't think it should be behind a #define, especially when it's longer to type the define out in most cases.